systemtests/src/main/java/com/github/streamshub/systemtests/utils/SetupUtils.java

```diff
@@ -27,8 +30,17 @@ public static String getYamlFileContent(String fileUrl) {
                 content.append(inputLine).append("\n");
             }
         } catch (IOException e) {
-            throw new RuntimeException("Cannot download YAML content from url: " + fileUrl, e);
+            throw new FileOperationException("Cannot download YAML content from url: " + fileUrl, e);
         }
         return content.toString();
```

Comment:
Just a nit, you can probably simplify this method by using `InputStream#readAllBytes`.
For example:
```java
try (var stream = new URL(fileUrl).openStream()) {
    return new String(stream.readAllBytes(), StandardCharsets.UTF_8);
} catch (IOException e) {
    throw new FileOperationException("Cannot download YAML content from url: " + fileUrl, e);
}
```